src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
+    }
+
+    @Override
+    public void stopWork() {
+        return "certain machines stopped working";
+    }
```

Comment:
The access modifier for the class 'Bulldozer' is missing. It's best practice to explicitly define the access level, even if it's the default. Consider whether this class should be public or have another access modifier.